lib/ansible/modules/storage/netapp/sf_snapshot_schedule_manager.py

```diff
@@ -0,0 +1,389 @@
+description:
+- Create, destroy, or update accounts on SolidFire
+
+options:
+
+    state:
+        required: true
+        description:
+        - Whether the specified schedule should exist or not.
+        choices: ['present', 'absent']
+
+    paused:
+        required: false
+        type: bool
+        description:
+        - Pause / Resume a schedule
+
+    recurring:
+        required: false
+        type: bool
+        description:
+        - Define if the schedule should recur
+
+    time_interval_days:
+        description: time interval in days
+        required: false
+        type: int
+        default: 1
+
+    time_interval_hours:
+        description: time interval in hours
+        required: false
+        type: int
+        default: 0
+
+    time_interval_minutes:
+        description: time interval in minutes
+        required: false
+        type: int
+        default: 0
+
+    name:
+        required: true
+        description:
+        - Name for the snapshot schedule
+
+    snapshot_name:
+        required: false
+        description:
+        - Name for the created snapshots
+
+    volumes:
+        required: false
+        type: list
+        description:
+        - Volume IDs that you want to set the snapshot schedule for.
+        - At least 1 volume ID is required for creating a new schedule.
+        - required when C(state=present)
+
+    retention:
+        required: false
+        description:
+        - Retention period for the snapshot.
+        - Format is HH:mm:ss
+
+    schedule_id:
+        required: false
+        description:
+        - The schedule ID for the schedule that you want to update or delete.
+
+    starting_date:
+        required: false
+        format: 2016--12--01T00:00:00Z
+        description:
+        - starting date for the schedule
+        - required when C(state=present)
+        - Please use two '-' in the above format, or you may see an error- TypeError, is not JSON serializable description
+'''
+
+EXAMPLES = """
+   - name: Create Snapshot schedule
+     sf_snapshot_schedule_manager:
+       hostname: "{{ solidfire_hostname }}"
+       username: "{{ solidfire_username }}"
+       password: "{{ solidfire_password }}"
+       state: present
+       name: Schedule_A
+       time_interval_days: 1
+       starting_date: 2016--12--01T00:00:00Z
+       volumes: 7
+
+   - name: Update Snapshot schedule
+     sf_snapshot_schedule_manager:
+       hostname: "{{ solidfire_hostname }}"
+       username: "{{ solidfire_username }}"
+       password: "{{ solidfire_password }}"
+       state: present
+       schedule_id: 6
+       recurring: True
+       snapshot_name: AnsibleSnapshots
+
+   - name: Delete Snapshot schedule
+     sf_snapshot_schedule_manager:
+       hostname: "{{ solidfire_hostname }}"
+       username: "{{ solidfire_username }}"
+       password: "{{ solidfire_password }}"
+       state: absent
+       schedule_id: 6
+"""
+
+RETURN = """
+
```

Comment:
not a blocker, but returning schedule information would be 'nice', schedule_id at the least when creating.